README.md

```diff
@@ -288,14 +301,14 @@ There are two different mode to generate the label for each route that the plugi
 
 In both modes by default the plugin generates a prefix using:
 
--   the [`fastify` prefix](https://www.fastify.io/docs/latest/Reference/Server/#prefix) used to register the plugin (`fastifyPrefix`)
--   the routes prefix passed in the plugin option `collect.routes.prefix` (`routesPrefix`)
+-   the [`fastify` prefix](https://www.fastify.io/docs/latest/Reference/Server/#prefix) used to register the plugin (normalized replacing `/` with `.`), we call it `fastifyPrefix`
+-   the routes prefix passed to the plugin option `routes.prefix`, we call it `routesPrefix`
 
 Generating a computed prefix like this:
 
 `<fastifyPrefix>.<routesPrefix>`
 
-##### `static`
+##### `static` mode
```

Comment:
I think it is so small, WYT if we increase the size?

Reply:
Yeah I think so too, but I wanted to have these section nested in the TOC (which is generated automatically by header size). I am not sure how to fix it atm.

Reply:
Ce can put `Routes labels generation mode` at the same level of `Configuration options` WDYT? Or we can remove substitute `API` directly with `Configuration options`

Reply:
🤔 I prefer the first solution

Reply:
ok :+1: